scripts/write-story-list.ts

```diff
@@ -52,7 +53,8 @@ async function main() {
   });
 
   // And write a file for the storybook-visual-regression-test `playwright` test to read.
-  fs.writeFileSync('./dist/docs/stories.json', JSON.stringify(storyIds, undefined, 2));
+  const storiesFileString = path.join('.', 'dist', 'docs', 'stories.json');
```

Comment:
```suggestion
  const storiesFilePath = path.join('.', 'dist', 'docs', 'stories.json');
```

Please rename in both files.

Reply:
fixed